use-case-guides/workday-request-time-off/README.md

```diff
@@ -3,129 +3,96 @@ design_pattern_id: 2
 
 ## Discovery Questions to Personalize Your Conversation  Design *(Customization Options)*
```

Comment:
Weird to use parentheses in a section header like this

Reply:
removed